test/contracts/LSP.spec.ts

```diff
@@ -85,7 +91,7 @@ describe("LSP", function () {
 
     const approvePromise = new Promise<void>((resolve) => {
       contracts.WETH.once(
-        contracts.WETH.filters.Approval(null, addresses.EMP),
+        contracts.WETH.filters.Approval(null, addresses.LSP),
```

Comment:
This should be `addresses.LSPCreator` I think

Reply:
Yes that is absolutely right.